CMakeLists.txt

```
@@ -42,7 +42,11 @@ configure_file(
   ${CCF_DIR}/src/host/version.h.in ${CCF_GENERATED_DIR}/version.h @ONLY
 )
 
-set(CONSENSUSES raft pbft)
+if(PBFT_TESTS)
+  set(CONSENSUSES raft pbft)
```

```suggestion
  set(CONSENSUSES "raft;pbft")
```

If we make this a list, we can use `IN_LIST` on it.

---

Ah, it seems this already creates a list, cool.